docker/dispatch.sh

```diff
@@ -1,13 +1,70 @@
 #!/usr/bin/env sh
 
+_term() {
```

Comment:
I think there's a couple issues with this script (and maybe was before too) which you can see highlighted with a shell checker. We should add it as part of the CI process and you can fix the issues it raises

Reply:
Huh, shellcheck really wants me to quote ints. I've applied the suggestions that don't break things and ignored the rest.